cms/djangoapps/contentstore/views/item.py

```diff
@@ -1021,16 +1024,22 @@ def _create_xblock_child_info(xblock, course_outline, graders, include_children_
                 child, include_child_info=True, course_outline=course_outline,
                 include_children_predicate=include_children_predicate,
                 parent_xblock=xblock,
-                graders=graders
+                graders=graders,
+                user=user
             ) for child in xblock.get_children()
         ]
     return child_info
 
 
-def _get_release_date(xblock):
+def _get_release_date(xblock, user=None):
     """
     Returns the release date for the xblock, or None if the release date has never been set.
     """
+    # If year of start date is less than 1900 then reset the start date to DEFAULT_START_DATE
+    if xblock.start.year < 1900 and user:
```

Comment:
@dan-f and @nedbat , I assume this is because `_update_with_callback` in L1041 takes `user` as an argument.